docs/Release Notes.md

```diff
@@ -1,8 +1,49 @@
 #Release Notes
 
-<!--January 1st, 2021-->
+<!--January 29th, 2021-->
 
 <details open>
+<summary><font size="+1"><b>January 29th, 2021</b></font></summary>
+<br>
+  <p class="new-button">New</p>
+  <ul>
+    <li>Public sharing of the reports created within any studio session is now available on Polly.</li>
+    <li>Added 14,727 new curated transcriptomics and metabolomics datasets with 9513 transcriptomics datasets being added to the LINCS Datalake.</li>
+  </ul>
+  <p class="update-button">Update</p>
+  <ul>
+    <li>Added specific error message to indicate presence of multiple groups with the same compound name in Labeled LC-MS Application.</li>
```

Comment:
I'm assuming this is the shiny app? Name is `Labeled LC-MS Workflow`. Also link the app page?